examples/source/multimedia-animations/Integrating_Videos_in_AMP_an_Overview.html

```diff
@@ -167,18 +167,19 @@
   </amp-youtube>
   <!-- ## What if your Video Provider is not Supported?-->
   <!--
-    Use `amp-iframe` for videos not supported in AMP.
-    Find more examples [here](/content/amp-dev/documentation/components/reference/amp-iframe-v0.1.md).
-  -->
-  <amp-iframe title="Video of Sintel, an independently produced short film"
-              width="500"
-              height="281"
-              layout="responsive"
-              sandbox="allow-scripts allow-same-origin allow-popups"
-              allowfullscreen
-              frameborder="0"
-              src="https://player.ooyala.com/iframe.html?ec=Vxc2k0MDE6Y_C7J5podo3UDxlFxGaZrQ&pbid=6440813504804d76ba35c8c787a4b33c&platform=html5">
-  </amp-iframe>
-
+    Use `amp-video-iframe` for videos not supported in AMP.
+    Find more examples [here](https://amp.dev/documentation/components/amp-video-iframe/).
+  -->
+  <amp-video-iframe id="myVideo"
+               src="https://manisaiprasad.github.io/Custom-video-player-with-AMP/"
```

Comment:
Can you please host the video iframe on amp.dev as well. You can add the file here: https://github.com/ampproject/amp.dev/tree/future/examples/static/samples/files and will be served from here: `/static/samples/files/your-file.html`.

Reply:
ok got it 👍🏻